keyboards/handwired/horrortroll/k552/keymaps/default/oled/oled_stuff.h

```diff
@@ -0,0 +1,75 @@
+/* Copyright 2021 HorrorTroll <https://github.com/HorrorTroll>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+// OLED animation
+#include "lib/bongocat.c"
+#include "lib/galaxy.c"
+#include "lib/wave.c"
+
+#ifdef OLED_ENABLE
+    oled_rotation_t oled_init_kb(oled_rotation_t rotation) {
```

Comment:
```suggestion
    oled_rotation_t oled_init_user(oled_rotation_t rotation) {
```

Reply:
I've forgot for leave this function in here, it is unused so I've deleted.